ngraph/frontend/tensorflow/src/frontend.cpp

```diff
@@ -12,20 +12,45 @@
 #include "op_table.hpp"
 #include "tf_framework_node.hpp"
 
-using namespace google;
-
-using namespace ngraph::frontend;
-using namespace ngraph::frontend::tf;
+using namespace ::ngraph::frontend;
+using namespace ::ngraph::frontend::tf;
 using namespace ::tensorflow::ngraph_bridge;
 
-using ::tensorflow::GraphDef;
-
 namespace {
```

Comment:
unnamed namespace?

Reply:
it means static function. it is used nowhere else.